src/System.IO.FileSystem.Watcher/src/System.IO.FileSystem.Watcher.csproj

```diff
@@ -63,6 +63,15 @@
       <Link>Common\Interop\Unix\Interop.IOErrors.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsOSX)' == 'true' ">
```

Comment:
More changes are necessary in this file.  We need to separate out of the current TargetsUnix section the files specific to the Linux implementation... those should be moved to a TargetsLinux section.